tests/nodescaling/scale_test.go

```diff
@@ -136,7 +141,7 @@ func (s *ScaleTestSuite) TestScale() {
 	}
 }
 
-func (s *ScaleTestSuite) TestScaleDynamicInput() {
+func (s *TFPScaleTestSuite) TestTFPScaleDynamicInput() {
```

Comment:
nit; test name and suite shouldn't have to share 'TFPScale' but we don't have established practices for this. It makes reading the test name in Qase / the jenkins job more complicated like this though and I'd like to avoid it if possible.

Reply:
I can change that, not a big deal.

Reply:
Resolved